STYLE_GUIDE.md

```diff
@@ -0,0 +1,42 @@
+# Style Guide

```

Comment:
I would have thought we'd keep only one file.   
if we go for a separate style guide,  let's move it unde ./docs